tgui/packages/tgui/interfaces/TraitorInfo.tsx

```diff
@@ -0,0 +1,249 @@
+import { useBackend, useLocalState } from '../backend';
+import { BlockQuote, Box, Section, Stack } from '../components';
+import { BooleanLike } from 'common/react';
+import { Window } from '../layouts';
+
+const allystyle = {
+  fontWeight: 'bold',
+  color: 'yellow',
+};
+
+const badstyle = {
+  color: 'red',
+  fontWeight: 'bold',
+};
+
+const goalstyle = {
+  color: 'lightblue',
+  fontWeight: 'bold',
+};
+
+const employerstyle = {
+  fontWeight: 'bold',
+};
+
+type Objective = {
+  count: number,
+  name: string,
+  explanation: string,
+  complete: BooleanLike,
+  uncompleted: BooleanLike,
```

Comment:
Bit weird, why are these two separate?

Reply:
they are two things: complete is whether the objective is done, uncompleted is if the objective was ever "undone"

Reply:
...any chance you can pick better names then?

Reply:
yeah sure